FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc

```diff
@@ -380,18 +331,18 @@ void CTPPSFastTrackingProducer::FastReco(int Direction,H_RecRPObject* station)
                     theta = CLHEP::pi - sqrt(thx*thx+thy*thy)*urad;
                     MatchCellId(cellId, recCellId_B, recTof_B, matchCellId, recTof); 
                 }
-                phi   = (Direction>0)?-atan2(thy,-thx):atan2(thy,thx); // defined according to the positive direction
-
+                //phi   = (Direction>0)?-atan2(thy,thx):atan2(thy,thx); // defined according to the positive direction
+                phi = atan2(thy,thx); // defined according to the positive direction
```

Comment:
@mundim , previous code seems to be more correct (I may be wrong). In any case, commented lines may be removed.

Reply:
I'll verify it carefully and remove the commented code.